internal/cmd/controller/gitops/reconciler/gitjob_controller.go

```diff
@@ -881,6 +882,10 @@ func argsAndEnvs(gitrepo *v1alpha1.GitRepo, debug bool, CACertsPathOverride stri
 			Name:  "HOME",
 			Value: fleetHomeDir,
 		},
+		{
+			Name:  fleetcli.FleetApplyConflictRetriesEnv,
+			Value: strconv.Itoa(fleetcli.GetOnConflictRetries()),
```

Comment:
Now we are converting it back?

Reply:
corev1.EnvVar expects a `string` and logically the number of retries is an `int` so we need to convert from the env value (`string`) to `int` when using the value in the retries loop or here.